source/app/viewoptions.h

```diff
@@ -32,7 +39,14 @@ class ViewOptions
     void clearFilter() { myFilter.reset(); }
 
     double getZoom() const { return myZoom; }
-    void setZoom(double percent) { myZoom = percent; }
+
+    /// Fixates the zoom level. Note that we support two types of zoom
+    /// changes - explicit level or one from a list
+    bool setZoom(double percent);
+    /// Increases to the next possible zoom level.
+    bool increaseZoom();
+    /// Increases to the previous possible zoom level.
+    bool decreaseZoom();
```

Comment:
I decided to make them Booleans in case we decide to add an "error" sound upon hitting a bound